rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "1.72.0"
+channel = "1.77.0"
```

Comment:
Is it truly necessary to bump the version for this PR? I'm not against it, just checking if it's needed or not.

Reply:
restored